pynfe/utils/__init__.py

```diff
@@ -18,9 +18,14 @@
 
 
 # @memoize
-def so_numeros(texto):
-    """Retorna o texto informado mas somente os numeros"""
-    return "".join(filter(str.isdigit, str(texto)))
+def so_numeros(texto) -> str:
+    """
+    Retorna o texto informado mas somente os numeros
+
+    :param texto: String ou Inteiro a ser analisada
+    :return: String somente com números
+    """
+    return ''.join(filter(str.isdigit, str(texto)))
```

Comment:
```suggestion
    return ''.join(c for c in texto if c.isdigit())
```

Reply:
However, it does not guarantee if it receives an int value or None, it returns an error. As with failed tests.